lib/screens/calendar/calendar.dart

```diff
@@ -623,6 +594,11 @@ class _EntriesSlot extends StatelessWidget {
     return (input is Fragment$CalendarDay$entries$$EpisodeCalendarEntry);
   }
 
+  bool isHappeningNow(String endStr, String stStr) {
```

Comment:
Kind of scary to use strings because you can pass in whatever you want and the order is not clear: `isHappeningNow("null", "test")`.
It would be better with `bool isHappeningNow({DateTime start, DateTime end})` which makes it clear that you want two valid dates, and it's clear which is which: `isHappeningNow(start: DateTime.parse(start), end: DateTime.parse(end));
But even better would be to pass the entry itself, like this:

```dart
  bool isHappeningNow(Fragment$Entry entry) {
```
(psuedocode, replace type with correct type)